index.html

```diff
@@ -817,6 +817,12 @@ <h2 id="securing-vcs-with-cose">Securing JSON-LD Verifiable Credentials with COS
                 A [=conforming COSE verifier implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to verify
                 [=conforming COSE documents=] that use this media type.
             </p>
+            <p>
+                When including [=verifiable credentials=] secured with COSE in [=verifiable presentations=]
+                as <a 
+                data-cite="VC-DATA-MODEL-2.0/#defn-EnvelopedVerifiableCredential">Enveloped Verifiable Credentials</a>, the credentials MUST be encoded using hexadecimal
+                as specified in [[RFC4648]].
```

Comment:
Hexadecimal isn't allowed in data URLs AFAIK.

Reply:
updated